clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -1502,6 +1509,12 @@ enum DeclCode {
   /// An ImplicitConceptSpecializationDecl record.
   DECL_IMPLICIT_CONCEPT_SPECIALIZATION,
 
+  // A decls specilization record.
+  DECL_SPECIALIZATIONS,
+
+  // A decls specilization record.
+  DECL_PARTIAL_SPECIALIZATIONS,
+
```

Comment:
The `DECL_LAST` also needs to be updated her.e

Reply:
Thanks for catching this! Done in https://github.com/llvm/llvm-project/commit/366daddfad9aa38ebb7d40055cf65f4ecb7dd6f9